admin/syslog-ng/Makefile

```diff
@@ -32,62 +39,69 @@ endef
 define Package/syslog-ng/conffiles
 /etc/syslog-ng.conf
 /etc/syslog-ng.d/
+/etc/scl.conf
 endef
 
 define Build/Configure
 	$(SED) 's,-I/usr/include,,' $(PKG_BUILD_DIR)/configure
 	$(Build/Configure/Default)
 endef
 
-CONFIGURE_ARGS += \
-  $(call autoconf_bool,CONFIG_IPV6,ipv6) \
-         --disable-dependency-tracking \
-         --disable-amqp \
-         --disable-tcp-wrapper \
-         --disable-glibtest \
-         --disable-mongodb \
-         --disable-java \
-         --disable-json \
-         --disable-python \
-         --disable-spoof-source \
-         --disable-sql \
-         --disable-linux-caps \
-	 --disable-smtp \
-	 --disable-redis \
-         --enable-prce \
-
-TARGET_CPPFLAGS += \
-  -I$(STAGING_DIR)/usr/include/eventlog
+CONFIGURE_ARGS +=  \
+	$(call autoconf_bool,CONFIG_IPV6,ipv6) \
+	--disable-dependency-tracking \
+	--enable-tcp-wrapper \
```

Comment:
Is there a specific reason for this? Other OSes like Fedora and Arch Linux have deprecated tcp-wrappers as it has had no updates for ~20 years. I also tried to do this here unsuccessfully.

Reply:
According to [mailing list from syslog-ng](https://lists.balabit.hu/pipermail/syslog-ng/2018-February/024317.html), there is someone, who is still using it.

Reply:
I was talking about OpenWrt specifically. Does it provide any needed functionality? I ask since I would like to get rid of tcp_wrappers in the tree.